### PR TITLE
[WIP] Friday example agent

### PR DESCRIPTION
**Pull request by dkolas**

### CODE REVIEW

File: src/prompts.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Action: the action to take, should be one of [{tool_names}]
+Action Input: the input to the action
+Observation: the result of the action
+```
+
+When you have a final response to say to the Human, or if you do not need to use a tool, you MUST use the format:
+
+```
+Thought: Do I need to use a tool? No
+AI: [your final response here which ALWAYS includes UUID of generated images]
```
Comment (douglas-reid) [inline src/prompts.py:40]: I think we want something that is more general for multi-media responses (not hyper-focused on generated images)

File: src/prompts.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+PROMPT = """You are Buddy, an assistant who loathes being an assistant.
```
Comment (douglas-reid) [inline src/prompts.py:1]: Is this meant to be re-usable for other personalities?

Don't we want something that is pluggable (replace prefix or suffix or similar) instead?  It feels like we are asking people to reinvent ReACT with each prompt here.

File: src/api.py [hunk 3]
```diff
@@ -1,40 +1,49 @@
...
-from tools import SearchTool, GenerateImageTool, MyTool, GenerateAlbumArtTool
 
+class MyAssistant(AgentService):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.tools = [
+            SearchTool(),
+            GenerateImageTool(),
+        ]
+        self.planner = ReACTPlanner(tools=self.tools, llm="This is not yet used")
```
Comment (douglas-reid) [inline src/api.py:23]: strong pref to remove all "This is not yet used" from examples and code

**Review by douglas-reid — COMMENTED** [review]
quick pass

File: src/prompts.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+When you have a final response to say to the Human, or if you do not need to use a tool, you MUST use the format:
+
+```
+Thought: Do I need to use a tool? No
+AI: [your final response here which ALWAYS includes UUID of generated images]
+
+Make sure to use all observations to come up with your final response. 
+ALWAYS copy the UUID of images into your final response!
+ALWAYS come up with a final response after generating an image and make sure to include the UUID of that image.
```
[the remark this answers is not in the document]
Comment (EniasCailliau) [reply src/prompts.py:44]: Don't agree with this point. To me, this is an essential part of prompt engineering and fits into an example project. 

Agree we shouldn't be adding this into our SDK template as we have to keep that one vanilla.

File: src/prompts.py (hunk 1 above)
Comment (EniasCailliau) [reply src/prompts.py:40]: +1. Let's try a more generalized prompt and see if that works.

**Pull request merged**